### ISSUE TITLE
Disappeared Bitcoins

### ISSUE THREAD

**Issue by Mekcukahell** [asgardex/asgard-exchange#349]
Good day.
thor1pt3dy9p5gqjemeurj93chu8pczxl6xa0n7xqdc - Thorhain address
bc1qdj83gzyguzn86n7ydv9paz8u8gc2fcw9recy6p - Bitcoin Address.

On 04/14/21 I replenished my BTC account in the amount of 0.1775 - which were successfully invested in the liquidity pool.

On April 18, 21, I replenished my BTC account in the amount of 0.2 BTC - which were "stuck" due to an overflowing pool.

Litecoin is also stuck.

I did everything according to the instructions from the administrator (https://app.asgard.exchange/deposit-sym-recovery)

Litecoin returned to the account.

But only half of Bitcoin returned.

In total, I sent 0.3775 BTC to my wallet, and at the moment I have only 0.2 BTC on my balance.
I no longer have any liquidity pools either. What to do? Where did they disappear to?

**Comment by stuartwk** [comment]
Hey @Mekcukahell, this was related to a protocol level bug. I'm asking the core devs about this and will get back to you.

**Comment by eode** [comment]
I think I'm having the same issue.  

* I did a symmetrical deposit via Asgardex Electron Wallet, but the RUNE weren't deposited.
* The BCH seemed to disappear.   After a while, some bug reporting (reported on the Asgardex Electron wallet), I was recommended by Butterfly on Telegram to use the deposit-sym-recovery on Asgardex Exchange. 
* My pool was found, and I submitted the transaction (ensuring I had 3 Rune as specified)
* [This transaction](https://www.blockchain.com/bch/tx/ca4264c946569bedcd96ce06a7fd5c29d14290d764e26040b771abe4c88683b2) occurred, and appears to be the correct amount.  However, the result didn't end up in my account, and I don't have enough understanding of the technical details to trace what *did* happen.
* What *seems* to have occurred is that the address it was supposed to go to got a miniscule amount, and the rest went back to the originating address.
